app/assets/stylesheets/views/_browse.scss

```diff
@@ -177,16 +177,22 @@
       }
 
       ul .active a {
-        background: $govuk-link-colour;
+        background: $govuk-brand-colour;
```

Comment:
Is this because the brand colour is used for backgrounds so will be tested for contrast, whilst the link colour might not be? Could be worth a comment to make sure that it's not changed in the future by someone well-meaning.

Reply:
No, it's because both colours translate to the same thing but semantically saying "I am highlighting this item because the user selected it previously" doesn't seem to be a case for using a link colour - :active and .active combined with a variable called link-colour was overly confusing.